src/cgroups/blkio.rs

```diff
@@ -140,7 +140,7 @@ mod tests {
     fn setup(testname: &str, throttle_type: &str) -> (PathBuf, PathBuf) {
         let tmp = create_temp_dir(testname).expect("create temp directory for test");
         let throttle_file = set_fixture(&tmp, throttle_type, "")
-            .expect(&format!("set fixture for {}", throttle_type));
+            .unwrap_or_else(|_| panic!("set fixture for {}", throttle_type));
```

Comment:
What is the reason for using and unwrap_or_else for panicking instead of using expect? Shouldn't these be basically the same thing?

Reply:
@tsturzl According to cargo clippy, it's probably to prevent the format from being called every time.
https://rust-lang.github.io/rust-clippy/master/#expect_fun_call

Reply:
If you have no problem with this PR, please approve.